test/fshtypes/FshQuantity.test.ts

```diff
@@ -64,7 +64,7 @@ describe('FshQuantity', () => {
       const code = new FshCode('bar', 'http://foo.com', 'Display');
       const quantity = new FshQuantity(100, code);
       const result = quantity.toString();
-      expect(result).toEqual('100 \'bar\' "Display"');
+      expect(result).toEqual('100 http://foo.com#bar "Display"');
```

Comment:
I can't comment on the line I want, but on 61-62 there is a comment that looks like it is no longer true, so we can take that out.

Reply:
Oh there is also the same comment on 38-39 that can go.